brainiak/utils/fmrisim.py

```diff
@@ -771,7 +781,19 @@ def convolve_hrf(stimfunction,
                  scale_function=True,
                  temporal_resolution=1000.0,
                  ):
-    """ Convolve the specified hrf with the timecourse
+    """ Convolve the specified hrf with the timecourse. In order
+    to accelerate the (slow) convolution, stimfunction is downsampled (mean
+    filtering) by the specified temporal resolution (e.g. a stimfunction of
+    100000 elements becomes 100 elements if the temporal_resolution is
+    1000). If temporal_resolution is 1 then the output will be the same
+    length as stimfunction.
+    Be aware that if scaling is on and events are very short
+    (> temporal_resolution * tr_duration) then the hrf may or may not come
+    out as anticipated. This is because very short events would evoke a small
+    absolute response after convolution  but if there are only short events
+    and you scale then this will look identical to a convolution with longer
+    events. In general scaling is useful, which is why it is the default,
+    but be aware of this edge case
```

Comment:
Sorry I forget about this. Can user avoid this issue by setting scaling to false?

Reply:
Yes they can, I added a comment to say as much